src/App.js

```diff
@@ -1,25 +1,48 @@
 import React from 'react';
 import './App.css';
-import Counter from './components/Counter';
+import Navbar from "react-bootstrap/Navbar"
+import { HiArrowSmLeft } from "react-icons/hi"
+import { BrowserRouter as Router, Routes, Route ,useNavigate} from 'react-router-dom';
+import { useDispatch, useSelector } from "react-redux";
+import Home from "./components/Home"
+import Details from "./components/Details"
+import {  toggleBackButton } from "./redux/cases/cases";
+
+const NavigationBar = () => {
+  const { backButton } = useSelector(state => state);
+  const dispatch = useDispatch();
+  const history = useNavigate();
+  const currentStyle = {display: "none"};
+  if (backButton) {
+    currentStyle.display = "block";
+  } else {
+    currentStyle.display = "none";
+  }
+  return (
+    <Navbar className="navbar" variant="dark">
+        <HiArrowSmLeft style={currentStyle} className="navbar-brand" size={40} onClick={() => {history(-1); dispatch(toggleBackButton(toggleBackButton()))}}/>
+        <Navbar.Brand>
+          Covid-19 Tracker
+        </Navbar.Brand>
+      </Navbar>
+    )
+  }
+  
+
 
 const App = () => (
-  <main className="bg-dark text-white min-vh-100 d-flex flex-column justify-content-center">
-    <div className="container-md">
-      <div className="row mx-0 justify-content-center">
-        <div className="col-md-9">
-          <h1 className="display-2 text-center">Hello React-Redux!</h1>
-          <p className="lead">
-            This project will allow you to set up React and Redux in any other project in seconds.
-            All you have to do is download this repository&apos;s files, copy and paste them into
-            your other project&apos;s folder, run <code>npm install</code>, and voilá! Your project
-            is ready to run <code>npm start</code> for development build and{' '}
-            <code>npm run build</code> for production build.
-          </p>
-          <Counter />
-        </div>
-      </div>
-    </div>
+  <main>
+    <Router basename={process.env.PUBLIC_URL}>
+    <NavigationBar />
+    
+      <Routes>
+        <Route exact path="/" element={<Home />} />
+        <Route path="/details/:countryId" element={<Details />}/>
+        
```

Comment:
- [ ] The application fails to start it throws an error: `Error: Invalid date: Invalid Date`, kindly fix it.